reader.go

```diff
@@ -301,6 +301,13 @@ func decodeLineOfMasterPlaylist(p *MasterPlaylist, state *decodingState, line st
 				state.variant.Captions = v
 			case "NAME":
 				state.variant.Name = v
+			case "FRAME-RATE":
+				var val float64
```

Comment:
@cnguyenWurl - we don't need to declare val here. `val, err := strconv.ParseFloat(v, 64)` it already declares it here and will initialize it to 0.0 as the default. Also we are returning on err so it should work. Basically we can remove this line `var val float64`